genesis/example/wallet_test.go

```diff
@@ -0,0 +1,205 @@
+/*
+ *    Copyright 2018 INS Ecosystem
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package example
+
+import (
+	"testing"
+
+	"github.com/insolar/insolar/genesis/model/class"
+	"github.com/insolar/insolar/genesis/model/domain"
+	"github.com/insolar/insolar/genesis/model/object"
+	"github.com/insolar/insolar/genesis/model/resolver"
+	"github.com/satori/go.uuid"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewWalletDomain_WithNilParent(t *testing.T) {
+	_, err := newWalletDomain(nil)
+	assert.EqualError(t, err, "parent must not be nil")
+}
+
+func TestNewWalletDomain(t *testing.T) {
+	parent := &mockParent{}
+	proxy, err := newWalletDomain(parent)
+
+	assert.NoError(t, err)
+	assert.Equal(t, proxy,
```

Comment:
In `assert.Equal` second param is expected value, and third - actual value. And fix indents (last should be at next line).